runtime/ms-rest-azure/package.json

```diff
@@ -22,11 +34,12 @@
     "@types/node": "^7.0.10"
```

Comment:
min dependency for ms-rest should go to ^2.2.2 as the `_withAuthFile()` function in login.js depends on a function homeDir() added in msRest in this PR. After the CI for first commit is green will update this dep in second commit. until the ms-rest version is not published this PR will fail in the second commit due to this reason.